src/Ccap/Codegen/DbSupportType.purs

```diff
@@ -76,6 +83,7 @@ dbSupportTypes =
     , instances:
         { equal: "gov.wicourts.common.instances.uuid.equalUUID"
         , meta: "gov.wicourts.common.Meta.metaUuid"
+        , read: Just "gov.wicourts.common.instances.uuid.ReadInstances._"
```

Comment:
Unsure why this was necessary atm since there's a meta instance for UUID, but looking more

Reply:
Adding a get instance for UUID seems to make this compile with Read[Option[Instance][] dropped